### PR TITLE
feat: adapt new wolf gateway

### PR DESCRIPTION
**Pull request by azayz**
---

- [x] This PR references an open issue
- [x] Test added
- [x] Documentation adjusted

### CODE REVIEW

File: now/common/flow/flow-external-clip.yml [hunk 1]
```diff
@@ -5,10 +5,11 @@ with:
     JINA_LOG_LEVEL: DEBUG
 jcloud:
   version: '3.10.0'
-  gateway:
-    expose: false
   labels:
     team: now
+gateway:
+  jcloud:
+    expose: false
```
Comment (violenil) [inline now/common/flow/flow-external-clip.yml:12]: Should we make this change to other flow.yml as well, eg. for text-to-text app?

Comment (azayz) [reply now/common/flow/flow-external-clip.yml:12]: I can't find this in any other Flow yawls, gateway is only mentioned in external clip

File: now/constants.py [hunk 2]
```diff
@@ -63,7 +63,7 @@ class ModelDimensions(BetterEnum):
     'local': ('CLIPOnnxEncoder/latest', 'ViT-B-32::openai', ModelDimensions.CLIP),
     'remote': ('CLIPOnnxEncoder/latest-gpu', 'ViT-B-32::openai', ModelDimensions.CLIP),
 }
-EXTERNAL_CLIP_HOST = 'encoderclip-climbing-lobster-885354f905.wolf.jina.ai'
+EXTERNAL_CLIP_HOST = 'encoderclip-knowing-lamprey-e00c3526d1.wolf.jina.ai'
```
Comment (florian-hoenicke) [inline now/constants.py:66]: why is a new clip host required?

File: now/common/flow/flow-external-clip.yml [hunk 3]
```diff
@@ -5,10 +5,11 @@ with:
     JINA_LOG_LEVEL: DEBUG
 jcloud:
   version: '3.10.0'
-  gateway:
-    expose: false
   labels:
     team: now
+gateway:
```
Comment (florian-hoenicke) [inline now/common/flow/flow-external-clip.yml:10]: I think the video flow yaml and the high performance video flow yaml also has to be updated

**Review by florian-hoenicke — CHANGES_REQUESTED** [review]

**Comment by florian-hoenicke** [comment]
Great initiative pushing the adaption forward

**Review by violenil — APPROVED** [review]
LGTM!

File: now/common/flow/flow-external-clip.yml (hunk 3 above)
Comment (azayz) [reply now/common/flow/flow-external-clip.yml:10]: no because there are no gateway jcloud args specified

**Pull request merged**